### DISCUSSION TITLE
What is the meaning of series?

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#8427] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56187131/f41c32ec-ba3a-4fb2-95ab-321d43497f61)

**Comment by ABA2396** [comment, marked as answer]
After the official update, the "series" function has been added, which allows for consecutive battles for n times and unified settlement.
This feature hasn't been released on other client yet, so consider hiding it in future version updates.
